packages/image_picker/tizen/src/image_picker_tizen_plugin.cc

```diff
@@ -0,0 +1,310 @@
+          break;
+      }
+    }
+    SendResultWithError("Invalid permission");
+#else
+    PickContent();
+#endif
+  }
+  void PickContent() {
+    app_control_h handle = NULL;
+    int ret = app_control_create(&handle);
+    RET_IF_ERROR(ret);
+
+    ret = app_control_set_operation(handle, APP_CONTROL_OPERATION_PICK);
+    RET_IF_ERROR(ret);
+
+    ret = app_control_set_mime(handle, "image/*");
+    RET_IF_ERROR(ret);
+
+    ret = app_control_send_launch_request(handle, PickImageReplyCb, this);
+    RET_IF_ERROR(ret);
+
+    app_control_destroy(handle);
+  }
+
+#ifndef TV_PROFILE
+  static void AppRequestResponseCb(ppm_call_cause_e cause,
+                                   ppm_request_result_e result,
+                                   const char *privilege, void *data) {
+    ImagePickerTizenPlugin *plugin = (ImagePickerTizenPlugin *)data;
+    assert(plugin);
+
+    if (cause == PRIVACY_PRIVILEGE_MANAGER_CALL_CAUSE_ERROR) {
+      LOG_ERROR("app_request_response_cb failed! [%d]", result);
+      plugin->SendResultWithError("Invalid permission");
+      return;
+    }
+
+    plugin->PickContent();
+  }
+#endif
+
+  static void PickImageReplyCb(app_control_h request, app_control_h reply,
+                               app_control_result_e result, void *user_data) {
+    LOG_INFO("PickImageReplyCb called: %d", (int)result);
+
+    ImagePickerTizenPlugin *plugin = (ImagePickerTizenPlugin *)user_data;
+    assert(plugin != nullptr);
+    assert(plugin->result_ != nullptr);
+
+    if (result != APP_CONTROL_RESULT_SUCCEEDED) {
+      plugin->SendResultWithError("Not Found Images");
+      return;
+    }
+
+    char **value = NULL;
+    int count = 0;
+    int ret = app_control_get_extra_data_array(reply, APP_CONTROL_DATA_SELECTED,
+                                               &value, &count);
+    if (ret != APP_CONTROL_ERROR_NONE) {
+      plugin->SendResultWithError(std::to_string(ret),
+                                  appControlErrorToString(ret));
+      return;
+    }
+
+    if (count == 1) {
+      LOG_INFO("image path: %s", value[0]);
+      std::string src_path = value[0];
+      std::string dst_path;
+      if (plugin->imageResize_.Resize(src_path, dst_path)) {
+        plugin->SendResultWithSuccess(dst_path);
+      } else {
+        plugin->SendResultWithSuccess(src_path);
+      }
+      free(value[0]);
+    } else {
+      plugin->SendResultWithError("Not Found Images");
+    }
+
+    if (value) {
+      free(value);
+    }
+  }
+
+  void SendResultWithSuccess(std::string imagePath) {
+    if (result_ == nullptr) {
+      return;
+    }
+    result_->Success(flutter::EncodableValue(imagePath));
+    result_ = nullptr;
+  }
+  void SendResultWithError(std::string errorCode,
+                           std::string errorMessage = "") {
+    if (result_ == nullptr) {
+      return;
+    }
+    result_->Error(errorCode, errorMessage);
+    result_ = nullptr;
+  }
+
+  ImageResize imageResize_;
```

Comment:
For naming convention?
```suggestion
  ImageResize image_resize_;
```

Reply:
done.